### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by HtetNaing0814**
Linters added. My first portfolio project. For mobile nav only for now.

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @HtetNaing0814 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that does not have linter checks. you can see how to setup linters [here](https://github.com/microverseinc/linters-config/tree/master/html-css)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by HtetNaing0814** [comment]
Thank u so much. I'll do so. But I'd like to know how do I tag you?

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request  ❌

Hi @HtetNaing0814, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have linters setup correctly, doesn't have a pull request description [here](https://github.com/HtetNaing0814/Portfolio/pull/1#issue-1248146496) and make sure to add a descriptive pull request title.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request  ❌

Hi @HtetNaing0814, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have linters setup correctly which means i cant see the green check marks, doesn't have a pull request description [here](https://github.com/HtetNaing0814/Portfolio/pull/1#issue-1248146496) which means it is empty and make sure to add a descriptive pull request title like your project name "Portfolio: Setup".

## If you are not able to understand kindly tag me as shown below.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by HtetNaing0814** [comment]
I do really have no idea what you are talking about. I thought I added
linters before submitting again. Furthermore, Do I need to make new pull
request?

**Comment by HtetNaing0814** [comment]
Somebody help me please.

**Comment by sja-thedude** [comment]
> I do really have no idea what you are talking about. I thought I added linters before submitting again.

@HtetNaing0814 You have to correctly set up linters, which means i can't see the green check marks. Also, your PR doesn't have a pull request description https://github.com/HtetNaing0814/Portfolio/pull/1#issue-1248146496 which means it is empty, kindly add a description of what changes you made in the project. Also, make sure to add a descriptive pull request title like your project name is "Feature", kindly replace it with "Portfolio: Setup".

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <section class="background">
+        <div class="toolbar">
+            <div class="welcome">Welcome</div>
+            <img class="menu" src="./assets/menu.svg" alt="menu">
+        </div>
+        <!-- <div style="display: flex; justify-content: center;">
+            <p class="margin-10px" style="cursor: pointer;">Portfolio</p>
+            <p class="margin-10px" style="cursor: pointer;">About</p>
+            <p class="margin-10px" style="cursor: pointer;">Contact</p>
+        </div> -->
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ] Remove the commented code from your project as it makes code hard to read.

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            </a>
+        </div>
+        <!-- <div class="center margin-top-25px">
+            <img style="cursor: pointer;" src="./assets/down-arrow.svg" alt="angellist">
+        </div>
+        <header style="padding-top: 25px;">My Recent Works</header style="padding-top: 25px;"> -->
+
+            <!-- <div class="card">
+                card
+            </div> -->
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ] Remove the commented code from your project as it makes code hard to read.

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+
+<body>
+    <section class="background">
+        <div class="toolbar">
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ] The container of the section should be a header tag

File: portfolio.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            <div class="welcome">Welcome</div>
+            <img class="menu" src="./assets/menu.svg" alt="menu">
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ] 1. The hamburger button should be on the right and has a `nav` tag

File: portfolio.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        </div>
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ] The content should be aligned with flexbox

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+            <a href="">
+                <img class="margin-10px" src="./assets/medium.svg" alt="medium">
+            </a>
+            <a href="">
+                <img class="margin-10px" src="./assets/github.svg" alt="github">
+            </a>
+            <a href="">
+                <img class="margin-10px" src="./assets/angellist.svg" alt="angellist">
+            </a>
+        </div>
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ] The container of the section should be a `section` tag

File: portfolio.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        <div class="toolbar">
+            <div class="welcome">Welcome</div>
+            <img class="menu" src="./assets/menu.svg" alt="menu">
+        </div>
+        <!-- <div style="display: flex; justify-content: center;">
+            <p class="margin-10px" style="cursor: pointer;">Portfolio</p>
+            <p class="margin-10px" style="cursor: pointer;">About</p>
+            <p class="margin-10px" style="cursor: pointer;">Contact</p>
+        </div> -->
+        <p class="intro primary-color">Hey there. I’m Rex.</p>
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ]  The main title (e.g., "I'm Marianne ...") is a `h1` tag

File: portfolio.html [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        <p class="intro secondary-color">I am a full stack developer.</p>
```
Comment (arslanbisharat) [inline portfolio.html]: - [ ] The subtitle (if existing) uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`

File: README.md [hunk 9]
```diff
@@ -1 +1,37 @@
...
+
+Built With
+- HTML
+- CSS
+
+Getting Started
+
+To get a local copy up and running:
+
+1. Clone this repository or download the zip folder:
```
Comment (arslanbisharat) [inline README.md:12]: - [ ] [OPTIONAL] Please add a live link of your project

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @HtetNaing0814, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @HtetNaing0814 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

- [OPTIONAL ] I am sorry for these was not mentioned by previous reviewers but its very important for your project. You should have Sementic Element in your project. That means you should have some thing like 
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**